--- templateToPCUIC.ml.orig	2018-11-07 11:42:27.000000000 +0100
+++ templateToPCUIC.ml	2018-11-07 11:42:47.000000000 +0100
@@ -95,7 +95,7 @@
 (** val trans_global : T.global_context -> global_decl list * t **)
 
 let trans_global _UU03a3_ =
-  ((trans_global_decls (fst _UU03a3_)), (snd _UU03a3_))
+  ((trans_global_decls (fst _UU03a3_)), (Obj.magic (snd _UU03a3_) : UGraph0.t))
 
 module TL = LiftSubst
 
